src/modules/configs/config.py

```diff
@@ -0,0 +1,53 @@
+from json import loads
+from typing import Union
+import os
+
+from jsonschema import validate
+
+from ..global_vars import global_vars
+
+
+class Conf:
+    """Class for parsing configuration files"""
+
+    def __init__(self, conf_path: Union[bool, str], conf_type: str):
+        self.conf_type = conf_type
+        self.conf_path = self.get_conf_path(conf_path)
+
+    def parse_conf(self) -> Union[bool, dict]:
+        conf = self.load_conf()
+        if self.validate_conf(conf) is None:
+            return conf
+        return False
+
+    def validate_conf(self, conf: dict) -> None:
+        if self.conf_type == "application":
+            schema_file = application
+        elif self.conf_type == "project":
+            schema_file = project
+        else:
+            schema_file = operator
+        return validate(instance=conf, schema=getattr(schema_file, self.conf_type))
+
+    def load_conf(self) -> dict:
+        with open(self.conf_path, "r", encoding="utf-8") as fh:
+            return loads(fh.read())
+
+    def get_conf_path(self, conf_path: Union[bool, str]) -> str:
+        if not conf_path:
+            conf_path = self.get_conf_here()
```

Comment:
After some consideration, I think this is dangerous actually, especially for application (main) conf - user could overwrite that and we have no permissions verification implemented yet. Please remove get_conf_here() for now, maybe we will bring it back later.